Gauge/Source/Time.hsc

```diff
@@ -71,6 +76,58 @@ withMetrics f = allocaBytes (sizeTimeRecord * 2) $ \ptr -> do
     getRecordPtr ptr2
     (,,) <$> pure a <*> peek ptr <*> peek ptr2
 
+#ifdef __GHCJS__
+data CTimespec = MkCTimespec CTime CLong
+
+instance Storable CTimespec where
+  sizeOf _ = 8
+  alignment _ = 4
+  peek p = do
+    s  <- peekByteOff p 0
+    ns <- peekByteOff p 4
```

Comment:
same here, `4` should be replaced by `sizeOf CTime`

Reply:
`sizeOf CTime` may not be correct offset in general because of alignment.